commands/MongodbMigrate.ts

```diff
@@ -39,7 +39,11 @@ export default class MongodbMigrate extends MigrationCommand {
       ),
     );
     if (duplicates.size > 0) {
-      throw new Error(`found duplicate migration file names`);
+      throw new Error(
+        `found duplicate migration file names: ${[...duplicates]
+          .map(({ name }) => name)
```

Comment:
I didn't realize it was an array of objects. Aren't they all different, even if they some share the same name property?

Reply:
They are different, in some case we could have multiple with the same name.

Reply:
A test would be very useful to confirm the implementation. What about moving the normalization and validation to the `getMigrationFiles` function or some other function called by it ?